### PR TITLE
ported to CircuitPython

### PR DESCRIPTION
**Pull request by mikeysklar**

### CODE REVIEW

File: DIY_Thermal_Light_Painting/DIY_Thermal_Light_Painting.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import board
+import busio as io
+import neopixel
+import adafruit_mlx90614
+
+# use bitbangio only with ESP8266
+# * does not support hardware i2c
+# * comment out busio above
+# * express boards can use also bitbangio, but they have i2c hardware built-in
+#import bitbangio as io
```
Comment (tannewt) [inline DIY_Thermal_Light_Painting/DIY_Thermal_Light_Painting.py]: This isn't needed. We'll use bitbangio internally on the ESP8266 automatically now.

Comment (ladyada) [reply DIY_Thermal_Light_Painting/DIY_Thermal_Light_Painting.py]: ooh i didnt know that, we'll clean that up on some old guides!

**Comment by tannewt** [comment]
Yup, we changed it with 2.x I think.

**Comment by mikeysklar** [comment]
@tannewt - I've removed the bitbagio comments. Is there anything else I need to do to resubmit for approval?

**Comment by tannewt** [comment]
Nope! I just try not to work on weekends. (and went camping last night.)

**Pull request merged**